testing/upload_validation.ts

```diff
@@ -142,7 +142,7 @@ export const Limits = {
   ColumnMatcherRegex: 300,
   NumColumnMatchersPerFormat: 10,
   NetworkDomainUrl: 253,
-  PermissionsBatchSize: 1000,
```

Comment:
Intentional?

Reply:
Yes I think we should change this as well. Double checking with some experts